.github/workflows/ci.yml

```diff
@@ -46,6 +46,10 @@ jobs:
         uses: crate-ci/typos@master
       - name: Lint dependencies
         uses: EmbarkStudios/cargo-deny-action@v1
+      - name: Check workflow files
+        uses: docker://rhysd/actionlint:latest
```

Comment:
Not sure its really useful to keep this always running. CI is changing not that often. A link to the online check tool might be more relevant.

Reply:
I think it's fine, it's not much of a resource heavy thing anyways.

Reply:
Perhaps gate it on the path - not sure I like the docker approach to running something like this, but it seems like it takes 0 seconds to run, so not too much of a concern I guess.

Reply:
There are other ways of using this as well: <https://github.com/rhysd/actionlint/blob/main/docs/usage.md>

The docker approach looked safer than yolo-downloading a script and running it so I went with that